### PR TITLE
Ac letter opener gem

### PR DESCRIPTION
**Pull request by acamargo213**

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
-  devise :database_authenticatable, :registerable, :confirmable, :recoverable, stretches: 12
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :confirmable, :trackable, :validatable
+  devise :database_authenticatable, :registerable, :recoverable, stretches: 12
```
Comment (aangott) [inline app/models/user.rb]: How about de-duplicating these two lines?  Ideally we'd have one line that starts with `devise`, followed by a unique list of all the modules we want.

File: config/environments/development.rb [hunk 2]
```diff
@@ -55,4 +55,7 @@
   
  ## At this point, a number of instructions will appear in the console. Among these instructions, you'll need to set up the default URL options for the Devise mailer in each environment. Here is a possible configuration for 
 config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
-end
+
+# letter opener gem
+config.action_mailer.delivery_method = :letter_opener
```
Comment (aangott) [inline config/environments/development.rb]: Could you indent this line to make it more clear it is inside the block that ends on the following line?

File: app/views/devise/mailer/confirmation_instructions.html.erb [hunk 3]
```diff
@@ -1,5 +1,11 @@
...
+	<strong> Welcome <%= @email %>! </strong>
+</p>
 
 <p>You can confirm your account email through the link below:</p>
 
-<p><%= link_to 'Confirm my account', confirmation_url(@resource, confirmation_token: @token) %></p>
+<p>
+	<%= link_to 'Confirm my account', confirmation_url(@resource, confirmation_token: @token) 
+	<%= link_to @user.email
+		<button type="button" class="btn btn-primary">email</button>%>
```
Comment (aangott) [inline app/views/devise/mailer/confirmation_instructions.html.erb]: It looks like the `%>` tag at the end of line 10 needs to be at the end of line 9 instead.  It looks like this is causing an error when a user tries to create a new account.

**Review by aangott — CHANGES_REQUESTED** [review]
Hi Ale!  Good job getting devise and letter opener to work!  I have a few comments on this pull request.  Could you let me know when you've addressed them?  

Also, it's probably a good idea to merge master into your branch so you can get up to date with the changes Diana has made.

File: app/models/user.rb (hunk 1 above)
Comment (acamargo213) [reply app/models/user.rb]: It oddly doesn't show on my end?

**Review by aangott — APPROVED** [review]

**Pull request merged**